examples/java-matter-controller/java/src/com/matter/controller/commands/discover/DiscoverCommissionablesCommand.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ *   Copyright (c) 2023 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package com.matter.controller.commands.discover
+
+import chip.devicecontroller.ChipDeviceController
+import chip.devicecontroller.DiscoveredDevice
+import com.matter.controller.commands.common.CredentialsIssuer
+import com.matter.controller.commands.common.MatterCommand
+
+class DiscoverCommissionablesCommand(
+  controller: ChipDeviceController, credsIssuer: CredentialsIssuer?
+) : MatterCommand(controller, "commissionables", credsIssuer) {
+  private val MAX_DISCOVERED_DEVICES = 10
+
+  override fun runCommand() {
+    currentCommissioner().discoverCommissionableNodes()
+
+    // Pause for 7 seconds
```

Comment:
I would drop this comment  as it just restates the code.

Below, if supported, maybe `TimeUnit.SECONDS.sleep(7)` is more explicit.

Reply:
Also 7 is a bit hard-coded. May as well have a `private val TIME_TO_WAIT_FOR_RESULTS_MS` or `TIME_TO_WAIT_FOR_RESULTS_SECONDS` constant.

Reply:
Yeah I was curious about this too although it's existing code -- maybe in the future this can be refactored in a way that emits devices as they are discovered, rather than just waiting for 7 seconds to see what was found?

Reply:
https://github.com/project-chip/connectedhomeip/issues/25425